rpm/sailfish-secrets.spec

```diff
@@ -161,8 +172,19 @@ rm -rf %{buildroot}
 
 mkdir -p %{buildroot}/%{_docdir}/Sailfish/Secrets/
 mkdir -p %{buildroot}/%{_docdir}/Sailfish/Crypto/
+mkdir -p %{buildroot}/%{user_unitdir}/user-session.target.wants/
+mkdir -p %{buildroot}/%{_datadir}/dbus-1/services/
+mkdir -p %{buildroot}/%{_datadir}/mapplauncherd/privileges.d/
+
 cp -R lib/Secrets/doc/html/* %{buildroot}/%{_docdir}/Sailfish/Secrets/
 cp -R lib/Crypto/doc/html/* %{buildroot}/%{_docdir}/Sailfish/Crypto/
+install -m 0644 daemon/%{secretsdaemon}.service %{buildroot}/%{user_unitdir}
+install -m 0644 daemon/%{secretsdaemon}.privileges %{buildroot}/%{_datadir}/mapplauncherd/privileges.d/
+install -m 0644 daemon/org.sailfishos.secrets.daemon.discovery.service %{buildroot}/%{_datadir}/dbus-1/services/
+
+mkdir -p %{buildroot}/%{_datadir}/mapplauncherd/privileges.d/
```

Comment:
Copy/paste artifact? It's already done a few lines earler

Reply:
Thanks, fixed